testpilot/frontend/static-src/app/views/feedback-view.js

```diff
@@ -1,84 +1,45 @@
-import app from 'ampersand-app';
 import BaseView from './base-view';
```

Comment:
Ah yeah, okay... I missed that part of the prototype. Renaming the view to reflect it just being used for experiment disable might make things clearer